MovieMVP/DetailMovieModule/Presenter/DetailMoviePresenter.swift

```diff
@@ -13,6 +13,7 @@ final class DetailMoviePresenter: DetailMoviePresenterProtocol {
     var imageService: ImageServiceProtocol?
     var router: RouterProtocol
```

Comment:
```suggestion
    private var imageService: ImageServiceProtocol?
    private var router: RouterProtocol
```
пропустил, тут тоже сервисы заприватить